sys/drv/pci.c

```diff
@@ -148,9 +151,10 @@ void pci_bus_enumerate(device_t *pcib) {
           .owner = dev, .type = type, .flags = flags, .size = size, .rid = i};
 
         /* skip ISA I/O ports range */
-        rman_addr_t start = (type == RT_IOPORTS) ? (IO_ISAEND + 1) : 0;
+        bus_addr_t *startp = (type == RT_IOPORTS) ? &ioports_start : &mem_start;
```

Comment:
The pointer was here because I used the same variable to update the offset for either mmio resources of I/O ports. The code has been changed to me more readable and stop using pointer at this point.